location/templates/location.html

```diff
@@ -23,4 +24,38 @@
 {% block content %}
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
 {% leaflet_map "leaflet_map_main" %}
+    {% if object.review_set %}
+        <h2> Review: </h2>
+        <hr class="mt-0 mb-2 mt-2">
+        <div class="row h-65">
+            <div class="col">
+                <table class="table table-striped table-bordered table-hover table-sm results-table" width="100%">
+                    <thead>
+                        <tr>
+                            <th scope="col" class="th-sm">Review</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for review in object.review_set.all %}
+                        <tr>
+                            <td scope="row">
+                                {{ review.user }} @ {{ review.time }}  :
+                                <br><br>
+                                {{ review.content }}
+                            </td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    {% endif %}
+    <h2>Write some review</h2>
+    <form action="{% url "review" object.id %}" method="post">
+        {% csrf_token %}
+        {{ form }}
+        <input type="submit" value="Submit">
```

Comment:
Perhaps add `class="class="btn seccondaryAction btn-outline-primary"` for styling